src/app/views/query-response/adaptive-cards/AdaptiveCard.tsx

```diff
@@ -42,7 +42,7 @@ class AdaptiveCard extends Component<IAdaptiveCardProps> {
 
   public componentWillUnmount() {
     // Remove all references
-    delete this.adaptiveCard;
+    this.adaptiveCard = new AdaptiveCardsAPI.AdaptiveCard();
```

Comment:
The comment above say `Remove all references` but you're creating a new adaptiveCard reference. Why the disparity?

Reply:
the new linter refused to remove all references... Removing the comment